templates/concept.html

```diff
@@ -1,5 +1,5 @@
 {{#concept @this}}
-<section data-trackable="concept" aria-label="{{name}}" {{#if responsiveGrids}}data-o-grid-colspan="{{responsiveGrids}}"{{/if}} class="topic-card__myft {{#unless items.length}}topic-card--empty{{/unless}}{{#if lazyLoadImages}} topic-card--lazy-loading-images{{/if}} {{classes}}">
+<section data-trackable="concept" aria-label="{{name}}" {{#if responsiveGrids}}data-o-grid-colspan="{{responsiveGrids}}"{{/if}} class="topic-card__myft {{#unless items.length}}topic-card--empty{{/unless}}{{#if lazyLoadImages}} topic-card--lazy-loading-images{{/if}} {{classes}}" data-prioritised="{{#if prioritised}}true{{else}}false{{/if}}">
```

Comment:
As data attributes are boolean attributes could this just be reduced to this `{{#if prioritised}}data-prioritised{{/if}}`